Source/VCGeneration/Split.cs

```diff
@@ -1071,6 +1066,169 @@ bool ContainsAssert(Block b)
         return splits;
       }
 
+      public static List<Split> FocusImpl(Implementation impl, Dictionary<TransferCmd, ReturnCmd> gotoCmdOrigins, VCGen par)
+      {
+        bool IsFocusCmd(Cmd c) {
+          return c is PredicateCmd p && QKeyValue.FindBoolAttribute(p.Attributes, "focus");
+        }
+        List<Block> GetFocusBlocks(List<Block> blocks) {
+          return blocks.Where(blk => blk.Cmds.Where(c => IsFocusCmd(c)).Count() != 0).ToList();
+        }
+        var dag = Program.GraphFromImpl(impl);
+        var topoSorted = dag.TopologicalSort().ToList();
+        // If reallyFocus is set to true,
```

Comment:
The comment here and on line 1097 are repetitive.  Please consolidate in one place.